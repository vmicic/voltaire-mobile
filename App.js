import React from 'react';

import Navigation from './navigation/Navigation'

import { useEffect, useState } from 'react';
import { Text, View, ActivityIndicator } from 'react-native';
import { createStackNavigator } from '@react-navigation/stack';
import { NavigationContainer } from '@react-navigation/native';
import AsyncStorage from '@react-native-async-storage/async-storage';
import axios from 'react-native-axios/lib/axios';

export default function App() {


  return (
    <Navigation />
  );
}
