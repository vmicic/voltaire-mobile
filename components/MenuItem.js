import React from 'react'
import { StyleSheet, Text, View } from 'react-native';

export default function MenuItem({ menuItem }) {
    return (
        <View style={styles.menuItem}>
            <Text>
                {menuItem.name}
            </Text>
            <Text>
                {menuItem.description}
            </Text>
            <Text>
                {menuItem.price} RSD
            </Text>
        </View>
    )
}

const styles = StyleSheet.create({
    menuItem: {
        padding: 16,
        marginTop: 16,
    }
});