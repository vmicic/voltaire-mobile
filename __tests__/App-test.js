/**
 * @format
 */

import 'react-native';
import React from 'react';
import App from '../App';

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';
import MenuItem from '../components/MenuItem';

let menuItem = {
  name: 'da',
  price: '200',
  description: 'hej',
};

it('renders correctly', () => {
  const tree = renderer.create(<MenuItem menuItem={menuItem} />).toJSON();
  expect(tree).toMatchSnapshot();
});
