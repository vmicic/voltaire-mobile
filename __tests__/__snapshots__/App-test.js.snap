// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "borderBottomColor": "#e6ebe7",
      "borderBottomWidth": 1,
      "marginTop": 16,
      "padding": 20,
    }
  }
>
  <Text
    style={
      Object {
        "fontSize": 15,
        "fontWeight": "bold",
        "marginBottom": 5,
      }
    }
  >
    da
  </Text>
  <Text>
    hej
  </Text>
  <Text>
    200
     RSD
  </Text>
</View>
`;
